site/source/docs/porting/simd.rst

```diff
@@ -1015,3 +1015,154 @@ The following table highlights the availability and expected performance of diff
      - 💣 emulated with complex SIMD+scalar sequence
 
 Only the 128-bit wide instructions from AVX instruction set are available. 256-bit wide AVX instructions are not provided.
+
+
+====================================================== 
+Compiling SIMD code targeting ARM NEON instruction set
+======================================================
+
+Emscripten supports compiling existing codebases that use ARM NEON by
+passing the `-mfpu=neon` directive to the compiler, and including the
+header `<arm_neon.h>`.
+
+The following table highlights the availability of various intrinsics. 
```

Comment:
Does ❌ here mean "scalarized" like in the previous tables, or does it mean "unimplemented"? It would be good to be more explicit about that.

Reply:
Unimplemented. I'll try and be more explicit. Another thing to note is that at least half of all instructions are scalarized, as they are implemented for both 64 and 128 bit vectors [and 64-bit ones are obviously not compatible with wasm].

Reply:
Let me know what you think now.